static/js/draw.js

```diff
@@ -0,0 +1,446 @@
+    var PointArray=[];
+    var myOptions;
+    var map;
+    var creator;
+    var ShapeValue;
+    var PolygonPoints=[];
+    var Poly;
+    var curLatLng=[];
+    var creator;
+    var flag=0;
+
+    function initialise(){
+    	myOptions = {
+        zoom: 16,
+		center: new google.maps.LatLng(18.505599,73.822867),
+		mapTypeId: google.maps.MapTypeId.SATELLITE
+		}
+
+        map = new google.maps.Map(document.getElementById('main-map'), myOptions);
+		 
+		creator = new PolygonCreator(map);
+				 
+		ShapeValue=django.jQuery('#id_shape').val();
+		 
+		if(ShapeValue!="None") 
+        {
+            flag = 1;
+
+     	    var result = ShapeValue.substring(20,ShapeValue.length-2);
+
+    	    StringArray= result.split(",");
```

Comment:
update the code from line  31 to 46 (use two for loop , for split space and comma). use below code

var array= result.split(/[\s,]+/);
then start with line 47